android/java/org/chromium/chrome/browser/crypto_wallet/model/OnboardingViewModel.java

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.crypto_wallet.model;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+public class OnboardingViewModel extends ViewModel {
+    private final MutableLiveData<String> mPassword = new MutableLiveData<>();
+
+    public void setPassword(String password) {
+        mPassword.setValue(password);
```

Comment:
nit: might wanna use `postValue` for thread safety, currently it's always being called on main thread so no issue and could be intentional, feel free to resolve.